src/start_fact_frontend.py

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python3
 '''
     Firmware Analysis and Comparison Tool (FACT)
-    Copyright (C) 2015-2020  Fraunhofer FKIE
+    Copyright (C) 2015-2021  Fraunhofer FKIE
```

Comment:
Might I ask why you did not refactor this module akin to your refactoring of backend and db?

Reply:
because pylint did not object (there were already some "pylint: disable" comments)
changed and refactored common code to new base class